site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5790,6 +5790,11 @@ private function updateOverallComments(TaGradedGradeable $ta_graded_gradeable) {
 
     private function updateOverallComment(TaGradedGradeable $ta_graded_gradeable, $comment, $grader_id) {
         $g_id = $ta_graded_gradeable->getGradedGradeable()->getGradeable()->getId();
+        if ($comment === "") {
+            $this->deleteOverallComment($g_id, $grader_id, $ta_graded_gradeable->getGradedGradeable()->getGradeable()->isTeamAssignment());
```

Comment:
While this change looks good, I feel like I'm missing something on what the `deleteOverallComment` function does:

https://github.com/Submitty/Submitty/blob/05c533bde9a64ad48c37cb67c55f6b0bb6a83e1c/site/app/libraries/database/DatabaseQueries.php#L5825-L5833

I don't think we want to delete the entire gradeable_data row on removing the TA comment...?

Reply:
I changed it to remove the data in gd_overall_comment inside of gradeable_data; I might be missing it in the code but it looks like the gd_overall_comment column inside of gradeable_data is unused - would it be better to make a migration to also remove gd_overall_comment inside of gradeable_data?

Reply:
If I'm remember correctly, this is a deprecated column.  
Previously (a term or two ago) overall comment was not tied to a specific grader.
Now it is tied to a specific grader -- allowing us to have multiple overall comments from different graders and peer graders.
It is probably safe to remove at this point.  We could do it in this PR, or maybe make a separate PR?